### ISSUE TITLE
Electrum imposter site

### ISSUE THREAD

**Issue by partylana** [spesmilo/electrum#4713]
I'm seeing that ad since 10 days. 

<img width="799" alt="screen shot 2018-09-13 at 11 20 29 pm" src="https://user-images.githubusercontent.com/33454532/45506044-cb209180-b7ab-11e8-9421-645c1dbf53ca.png">

**Comment by bauerj** [comment]
This is very weird, that's actually the website of some kind of activity center for children in the Netherlands 🤔

**Comment by bauerj** [comment]
Aha!

Thanks, that redirection doesn't happen here. Not even from the Google ad which I see too.

We will contact the registrar and the hosting company which will hopefully lead to this site being taken down.

**Comment by partylana** [comment]
Yes its not happening now. Looks like the phisher looking at this thread. 🤣

**Comment by AbdussamadA** [comment]
the ad doesn't mention cryptocurrencies to bypass google's ban on crypto ads. very clever
